capa/features/extractors/ida/file.py

```diff
@@ -152,6 +152,8 @@ def extract_file_function_names():
         if idaapi.get_func(ea).flags & idaapi.FUNC_LIB:
             name = idaapi.get_name(ea)
             yield FunctionName(name), ea
+            if name.startswith("_"):
+                yield FunctionName(name[1:]), ea
```

Comment:
`name[1:]` vs `name.lstrip("_")`

one removes all prefixes, the other removes exactly one. can you confirm a final time this is what we want?

also, recommend including a comment here explaining what we're handling.

Reply:
I'm basing this updated logic on how functions are defined by MSDN. We want to remove exactly one `_` as there are functions defined with a leading `_` e.g. `_wfopen` (https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/fopen-wfopen?view=msvc-170). In the case of `_wfopen` the linker may prefix as `__wfopen`. Using the logic mentioned here we would emit both `_wfopen` and `__wfopen`. `wfopen` is not defined by MSDN therefore it should not be emitted.

Reply:
This assumes the `_` prefix is always added (my understanding).

Reply:
perfect, thanks!